frontend/gateway/forwarder/forward.go

```diff
@@ -149,6 +151,34 @@ func (c *bridgeClient) discard(err error) {
 	}
 }
 
+func (c *bridgeClient) NewContainer(ctx context.Context, req client.NewContainerRequest) (client.Container, error) {
+	ctrReq := gateway.NewContainerRequest{
+		ContainerID:  identity.NewID(),
+		NetMode:      req.NetMode,
+		SecurityMode: req.SecurityMode,
+	}
+
+	for _, m := range req.Mounts {
+		refProxy, ok := m.Ref.(*ref)
+		if !ok {
+			return nil, errors.Errorf("Unexpected Ref type: %T", m.Ref)
```

Comment:
nit: lowercase

couple of more cases like this. search for `errors.Errorf/Wrap*`

Reply:
Updated the casing on all the error messages I could find.